test/Api/ConstraintApiTest.php

```diff
@@ -1,112 +0,0 @@
-<?php
```

Comment:
Deleted all of the test code after I realised they were all empty shell files that did not test anything!